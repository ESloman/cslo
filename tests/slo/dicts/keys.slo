var map = {"a": 1, "b": 2, "c": 3};
print(map.keys());
for (var key in map.keys()) {
    print(key);
}
