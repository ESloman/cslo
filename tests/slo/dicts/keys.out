list[3]: [c, a, b]
c
a
b
